app/controllers/homes_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomesController < ApplicationController
+  def index
+    @posts = current_user.posts
+  end
```

Comment:
- [ ] Index displays posts in a timeline (recent to older). We can achieve this by using [default scope](https://apidock.com/rails/ActiveRecord/Base/default_scope/class) in `post.rb`. An example could be [this](https://stackoverflow.com/questions/3393687/default-sort-order-for-a-rails-model).